contracts/AnchoredPriceView.sol

```diff
@@ -288,6 +295,15 @@ contract AnchoredPriceView {
         revert("Unknown token address");
     }
 
+    function invalidate(bytes memory message, bytes memory signature) public {
+        (string memory decoded_message, address target) = abi.decode(message, (string, address));
+        require(keccak256(abi.encodePacked(decoded_message)) == keccak256(abi.encodePacked("rotate")), "invalid message must be 'rotate'");
+        require(priceData.source(message, signature) == source, "invalidation message must come from the reporter");
+        target; //shh
```

Comment:
curious what this line does?

Reply:
```
➜  open-oracle git:(cob/invalidation) yarn compile
yarn run v1.22.4
warning ../../package.json: No license field
$ npx saddle compile
Compiling contracts contracts/*.sol tests/contracts/*.sol with solc to /Users/coburn/Compound/open-oracle/.build/contracts.json...
Warning: Unused local variable.
   --> contracts/AnchoredPriceView.sol:299:41:
    |
299 |         (string memory decoded_message, address target) = abi.decode(message, (string, address));
    |                                         ^^^^^^^^^^^^^^


```
just to get rid of this unused local variable complaint : /

Reply:
wow 🤩

Reply:
I think on this one you could also do `(string memory decoded_message, ) = ...`

Reply:
perfect, I tried (string memory, address ) and that failed so I gave up. Your take works though